api/graphql/makeModels.js

```diff
@@ -218,6 +218,12 @@ export default function makeModels (userId, isAdmin) {
             type: filter,
             sort: sortBy,
             topic
+          }),
+        betaAccessCode: c =>
+          Membership.hasModeratorRole(userId, c.id)
```

Comment:
we might end up doing something more sophisticated re these kind of permissions in graphql. But for now this seemed fine.